xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
Hi @robhoes ,

I do not understand the semantics of the above code. If we have ipv4 setting like the following,

[("address", ["192.168.1.100/24", "192.168.1.200/24"]); ("gateway", ["192.168.1.1"])]

then it looks like the above ip_configuration type cannot represent the ipv4 setting.